### PR TITLE
Territory languages

### PR DESCRIPTION
**Pull request by artemShelest**
A territory-language relations based on data available from unicode.org list. This is not a real ISO standard/draft, just using the uniform handler to load.
A JSON db file can be updated with `python scripts/make_territory_languages.py` from the project directory.
As a further development, I can support reverse lookups to select territories, where particular language is used.

### CODE REVIEW

**Comment by artemShelest** [comment]
Should I update anything else?

**Comment by sokil** [comment]
Yes, see all my comments. And your test does not test you code.

**Comment by artemShelest** [comment]
Do not see any of your comments, except the above. None found in the "Files changed" as well. Can you, please, point me there?

File: src/Database/Territory/Territory.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        return $this->officialLanguages;
+    }
+
+    /**
+     * @return array
+     */
+    public function getUnofficialLanguages()
+    {
+        if ($this->unofficialLanguages === null) {
+            $this->officialLanguages = array_filter($this->getLanguages(), function ($v) {
```
Comment (sokil) [inline src/Database/Territory/Territory.php]: Still bug

**Comment by artemShelest** [comment]
I agree on issues.
In general, it would be better not to use your base and relate existing Language instances with Country instances with some mediator object.
Please, let me know how relevant to your project is this enhancement from your point of view. I decided not to use your code in my project, as found JSON versions of http://cldr.unicode.org/ (where I take territory/language dependencies for this pull request), which have translations as well. If you find this change to be a good improvement - I can work on it, otherwise I withdraw the request.

**Comment by sokil** [comment]
only if you have time and inspiration :)) PRs are always wellcome!

**Comment by artemShelest** [comment]
I'm closing this for now. Thank you for you port anyway.

**Pull request closed without merging**